packages/core/src/sass/app/_common/layout/app-layouts.scss

```diff
@@ -143,7 +143,7 @@
             margin-top: 0;
         }
         &--is-dtrader-v2 {
-            height: calc(100% - #{$MOBILE_HEADER_DTRADER_V2_HEIGHT});
+            height: calc(100% - #{$MOBILE_NAV_BUTTON_DTRADER_V2_HEIGHT});
```

Comment:
Is height calculated now based on bottom nav on purpose? I thought it should be based on $MOBILE_HEADER_DTRADER_V2_HEIGHT
Please, ignore if irelevant

Reply:
no no, you are right, I thought buttom-nav div is outside as the header, now I see button navigation is inside app-content, I will make the change